services/web/server/src/simcore_service_webserver/oas3/v0/openapi.yaml

```diff
@@ -0,0 +1,222 @@
+        default:
+          $ref: '#/components/responses/DefaultErrorResponse'
+  /auth/logout:
+    get:
+      responses:
+        '200':
+          description: Succesfully logged out
+          content:
+            application/json:
+              schema:
+                $ref: 'components/schemas/log_message.yml#/LogMessageEnveloped'
+        default:
+          $ref: '#/components/responses/DefaultErrorResponse'
+  #/auth/reset-password:
+  #/auth/change-email:
+  #/auth/change-password:
+  /auth/confirmation/{code}:
+    get:
+      operationId: auth_confirmation
+      parameters:
+        - name: code
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        default:
+          $ref: '#/components/responses/OK_NoContent_204'
+
+
+
+components:
+  responses:
+    # TODO:Envelope objects are still not well/easily defined. See discriminators
+    OK_NoContent_204:
+      description: everything is OK, but there is no content to return
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/log_message.yml#LogMessageEnveloped'
+
+    DefaultErrorResponse:
+      description: Unexpected error
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    DataError_BadRequest_400:
+      description: requested information is incomplete or malformed
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    DataError_UnprocessableEntity_422:
+      description: requested information is ok but invalid
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    DataError_NotFound_404:
+      description: everything is okay, but the resource doesnt exist
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    DataError_Conflict_409:
+      description: a confict of data exists, even with valid information
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    AuthError_Unauthorized_401:
+      description: access token isnt provided or is invalid
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    AuthError_Forbidden_403:
+      description: an access token is valid, but requires more privileges
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
+
+    InternalServerError_500:
+      description: server throws an error, completely unexpected
+      content:
+        application/json:
+          schema:
+            $ref: 'components/schemas/error.yml#/ErrorEnveloped'
```

Comment:
I think I would not bother defining each and different responses as they are in fine always the same. Also HTTP defines these code so why redefining them?